### ISSUE TITLE
Lost my content despite saving

### ISSUE THREAD

**Issue by Aliwasan** [marktext/marktext#3803]
### Description

I have lost a day of work in my file content despite the fact that I had saved it I don't understand the file content is as it was 2 days ago !! 

- [ ] Can you reproduce the issue? 
I was adding some more comment to my file just a moment ago then saved it again. Then,  I opened another md. file from another directory. But when I got back to the file I worked on, all the content I had added from yersterday was gone !!

### Steps to reproduce

1. Writing on my .md file
2. did cmd + S to save the file 
3. did not close the file.
4. Opened another .md file from another directory. 
5. clicked on the tab of my original file to copy some content : but all my comments from yesterday and this morning were gone !

**Expected behavior:**

**Actual behavior:**

I was adding some more comment to my file just a moment ago then saved it again. Then,  I opened another md. file from another directory. But when I got back to the file I worked on, all the content I had added from yersterday was gone !!

### Versions

- MarkText version: v0.17.1
- Operating system: MacOS

Is there any way I can get back my file as it was an hour ago ?

**Comment by hittjwX** [comment]
I also had this problem on Ubuntu. After a good 400 words, I switched to another tab. When I came back, my edits were gone. Autosave enabled. Another time similar to #3232

**Comment by vpmapelli** [comment]
This just happened to me! I am guessing that my content got cleared out since I opened a new tab by mistake, trying to switch to plain text via shortcut. Even though I saved this file yesterday, before closing, everything is gone. I got an empty file now.

In the config folder, I can still see the images I added. Is there any way to restore the content?

**Comment by hittjwX** [comment]
I stopped using MarkText. I was using it to editor feature articles and small reports. The tab functions are broken in MarkText.

These markdown are in a GitHub repos. I now edit in a Workspace in Zettlr. This gets me closer to pandoc to PDF.

**Comment by WelliSolutions** [comment]
I just had the same problem on Windows, using version 0.17.1. I noticed that a question mark was missing at the end of the file. Adding the question mark didn't even change the status (the green "modified" dot did not appear). After adding a space character at the end, that indicator appeared. After saving, it disappeared (as expected). However, there are no changes detected by Git. I am not using tabs and this is the only instance of MarkText that I have open.

**Comment by jaclas** [comment]
I have the same problem. I'm using Kubuntu 24 LTS and the latest version of Mark Text 0.17.1.
As long as one document (tab) is open in Mark Text, everything works ok. Opening a second and subsequent ones turns on readonly mode, although you can still write in the document, but clicking on another paragraph undoes the change in the previous one. 
Closing all tabs except the last one no longer fixes the problem, you have to close Mark Text completely and open it from scratch. 
In other words, Mark Text is a single file editor for the moment.
